ONLYOFFICE/Docker-DocumentServer
# Merge branch release/v7.2.0 into master

- Fix Bug 53170 / Add the ability to set secure_link_secret (#444)
- Add multiarch build with action (#445)
- Add short tags for released images (#446)
- Fix for redis correct work (#449)
- Update base image version (#447)
- fix Bug 50138 / Fix SSL key access error (#455)
- Refactoring workflow (#448)
- Fix supervisor socket messages (#459)
- Fix Big #57286 / Add redis password config settings (#466)
- Fix bug #58032 / Fix moves certificates alarm messages (#461)
- Add another tags processing (#464)
- Add the missing variable (#467)
- Enable JWT by default and add a JWT status message (#482)
- Fix command to get container id (#483)
- Add latest tag for develop build (#484)
- fix build (#486)
- Add build Documentserver without example (#473)
- Build: Set job fail status if some build failed (#488)
- Fix KylinOS start error (#471)
- Add expected build arch (#489)
- Refactoring stable build (#490)
- fix Bug 58778 - Correct the display of container ID for 22.04 (#494)
- Refactoring testing build (#496)
- Fix makefile (#498)
- Fix makefile docker image (#499)
